index.html

```diff
@@ -1,167 +1,249 @@
-                  <p>Challenge Two</p>
-                </button>
-              </a>
-            </div>
-            <div class='col-md-6 col-lg-6 col-sm-12 col-xs-12 mt-3'>
-              <a href='quiz/quiz.html'>
-                <button class='btn btn-default btn-block web-nav'>
-                  <img class='nav-thumbnail' src='home-page/img/thumbnails/quiz01.png' alt=''>
-                  <p>Quiz</p>
-                </button>
-              </a>
-            </div>
-            <div class='col-md-6 col-lg-6 col-sm-12 col-xs-12 mt-3'>
-              <a href='useful-links/useful-links.html'>
-                <button class='btn btn-default btn-block web-nav'>
-                  <img class='nav-thumbnail' src='home-page/img/thumbnails/links.png' alt=''>
-                  <p>Useful Links</p>
-                </button>
-              </a>
+      <div class="website-navigation">
+        <h1 class="section-header">Website Navigation</h1>
+        <div class="section-container">
+          <a href="challenges/challenge-1/challenge-1.html" class="web-nav">
+            <img
+              class="nav-thumbnail"
+              src="home-page/img/thumbnails/challenge01.png"
+              alt=""
+            />
+            <p>Challenge One</p>
+          </a>
+
+          <a href="challenges/challenge-2/challenge-2.html" class="web-nav">
+            <img
+              class="nav-thumbnail"
+              src="home-page/img/thumbnails/challenge02.png"
+              alt=""
+            />
+            <p>Challenge Two</p>
+          </a>
+
+          <a href="quiz/quiz.html" class="web-nav">
+            <img
+              class="nav-thumbnail"
+              src="home-page/img/thumbnails/quiz01.png"
+              alt=""
+            />
+            <p>Quiz</p>
+          </a>
+
+          <a href="useful-links/useful-links.html" class="web-nav">
+            <img
+              class="nav-thumbnail"
+              src="home-page/img/thumbnails/links.png"
+              alt=""
+            />
+            <p>Useful Links</p>
+          </a>
+        </div>
+        <h1 class="section-header">Widgets</h1>
+        <div class="container-fluid google-wrap">
+          <!-- Google Search Widget by Mamun Abdullah-->
+          <div class="widget row d-flex justify-content-center">
+            <div class="google-search-container">
+              <form id="googleSearch">
+                <div class="btn-group">
+                  <input
+                    id="searchMe"
+                    class="google-search"
+                    type="text"
+                    placeholder="Google Search"
+                  />
+                  <button type="submit" class="btn btn-lg btn-primary">
+                    Search
+                  </button>
+                </div>
+                <div class="btn-group-vertical">
+                  <input
+                    id="searchMe"
```

Comment:
I think this should only take up one line for consistency and space efficiency. I want to enforce consistent style so that it's easier for people to recognize patterns in the codebase.

Reply:
@DavidMatthewFraser 

do you mean the formatting? sorry i had the prettier extension on so this is the why it formats like this, i will use a different formatter on this project in future il change it now 😄 👍